projectHelpers.js

```diff
@@ -3,6 +3,10 @@ const fs = require("fs");
 const semver = require("semver");
 const { EOL } = require("os");
 
+const { getProjectData, PROJECT_DATA_GETTERS } = require("./nsCliHelpers");
+
+const APP_DIR = "app";
```

Comment:
You can reuse this constant [here](https://github.com/NativeScript/nativescript-dev-webpack/blob/2e9405615f71bba45ffcb242805141dbc10e3ac5/index.js#L14)